qa-covid19.planx-pla.net/portal/gitops.json

```diff
@@ -210,7 +210,16 @@
         ]
       },
       "table": {
-        "enabled": true
+        "enabled": true,
+        "fields": [
+          "code",
```

Comment:
`code` doesn't exist in `subject`, if you mean the project, you can use `project_id` (it will be displayed as "Dataset")

Reply:
Ah, got it. I went with code because it was going to look nicer JHU vs. open-JHU.